app/src/main/java/com/infomaniak/mail/utils/Utils.kt

```diff
@@ -50,6 +50,8 @@ object Utils {
 
     const val SCHEME_SMSTO = "smsto:"
 
+    const val PRIMARY_COLOR_CODE = "--kmail-primary-color"
```

Comment:
Wouldn't it make sense to put this in UiUtils instead ?

Reply:
Could very well be, indeed. I moved it there